src/fields/abstractField.js

```diff
@@ -29,8 +29,8 @@ export default {
 			cache: false,
 			get() {
 				let val;
-				if (isFunction(this.schema.get)) val = this.schema.get(this.model);
-				else if (this.model && this.schema.model) val = objGet(this.model, this.schema.model);
+				if (isFunction(objGet(this.schema, 'get'))) val = this.schema.get(this.model);
+				else val = objGet(this.model, this.schema.model);
```

Comment:
I know that @icebob like to use bracket-less `if/else`, but I prefer to use them. It is more readable, easier to update later if we need more lines and we have the machine to do the minification already.
I know this is opinion territory so it is not blocking the merge, just my two cent.
Everything else is good for me 👍

Reply:
yeah, I prefer the bracket conditionals myself ... in this case, I just modified the existing lines quickly... adding the brackets in now, so they are there in the future.

Reply:
Yay team bracket ! :fist: